### PR TITLE
netcoreapp5.0 -> net5.0

### PR DESCRIPTION
**Pull request by baronfel**
This cherry-picks @thinkbeforecoding's fix for net50 to the bugfix branch so that the 5.x stables can use net5.0 as well.

### CODE REVIEW

**Comment by forki** [comment]
I tried to release this but seems the certificate for signing paket.exe is outdated. paging @agross

**Comment by forki** [comment]
This is now released

**Comment by baronfel** [comment]
excellent! thank you for running this down :)

File: src/Paket.Core/Versioning/FrameworkHandling.fs [hunk 1]
```diff
@@ -658,6 +656,7 @@ type FrameworkIdentifier =
         | DotNetFramework FrameworkVersion.V4_7_1 -> [ DotNetFramework FrameworkVersion.V4_7; DotNetStandard DotNetStandardVersion.V2_0 ]
         | DotNetFramework FrameworkVersion.V4_7_2 -> [ DotNetFramework FrameworkVersion.V4_7_1 ]
         | DotNetFramework FrameworkVersion.V4_8 -> [ DotNetFramework FrameworkVersion.V4_7_2 ]
+        | DotNetFramework FrameworkVersion.V5 -> [ DotNetCoreApp DotNetCoreAppVersion.V3_1; DotNetStandard DotNetStandardVersion.V2_1 ]
```
Comment (forki) [inline src/Paket.Core/Versioning/FrameworkHandling.fs:659]: should we add DotNetFramework FrameworkVersion.V4_8 as compat here?

Comment (thinkbeforecoding) [reply src/Paket.Core/Versioning/FrameworkHandling.fs:659]: This is something that is not clear for now... net5.0 should implement all netcoreapp3.1, but it definitely don't implement all net48 (remoting/messaging/etc...). So I'd say no.

Comment (forki) [reply src/Paket.Core/Versioning/FrameworkHandling.fs:659]: thanks! so we keep it as is.

**Pull request merged**